pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -90,9 +92,16 @@ func (h *mtuHandler) Init(_ context.Context) error {
 		return errors.Wrap(err, "error initializing iptables")
 	}
 
+	h.tableType = h.pFilter.GetMSSClampTable()
+	h.chainType = packetfilter.ChainTypeRoute
+
+	if h.tableType == packetfilter.TableTypeFilter {
+		h.chainType = packetfilter.ChainTypeFilter
+	}
```

Comment:
Here the API tells which table type to use but the user has to figure out the matching chain type. How about we modify `GetMSSClampTable` to also return the chain type:

  `GetMSSClampInfo() (TableType, ChainType)`

Reply:
Sounds good, I'll change it.